packages/idx/src/idx.ts

```diff
@@ -772,6 +774,7 @@ const angularJsService = (
             instanceLink.Search[uid] = []
         }
         if (listUid) {
+            console.log('added', uid, 'to', listUid, 'instanceLink')
```

Comment:
I'm not sure if you're still testing this line, but the console log is still running here.  :)

Reply:
Thanks, it wasn't intended to add the debugging of the deprecated logic

Reply:
Sweet!